components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -50,10 +54,14 @@ export class CreateObjectModalComponent implements OnInit {
   }
 
   createObject(): void {
-    this.createClicked.emit();
+    this.createClicked.emit(this.projectsSelected);
   }
 
   private isNavigationKey(event: KeyboardEvent): boolean {
     return event.key === 'Shift' || event.key === 'Tab';
   }
+
+  projectsDropdownAction(projects) {
```

Comment:
I was going to say this should be typed like so...
```suggestion
  projectsDropdownAction(projects: Project[]): void {
```
But what you have here are _not_ full `Project` objects--they lack a type field.
But you then proceed (L65) to treat them as full Project objects in any case.
So please check that the type field is correctly getting filled in somewhere, too.